File: inspect_db.py
import sqlite3
conn = sqlite3.connect('src/crm/agents.db')
cursor = conn.cursor()
cursor.execute("SELECT name FROM sqlite_master WHERE type='table';")
tables = cursor.fetchall()
print("Tables:", tables)

for table in tables:
    table_name = table[0]
    cursor.execute(f"PRAGMA table_info({table_name});")
    print(f"Schema for {table_name}:", cursor.fetchall())
    
conn.close()

File: inspect_db.ts
import Database from 'better-sqlite3';

const db = new Database('src/crm/agents.db');
const tables = db.prepare("SELECT name FROM sqlite_master WHERE type='table'").all();
console.log('Tables:', tables);

for (const table of tables) {
    try {
        const columns = db.prepare(`PRAGMA table_info(${table.name})`).all();
        console.log(`Table ${table.name} columns:`, columns);
    } catch (e) {
        console.error(e);
    }
}
db.close();
